resources/js/mp-history/game.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import StringWithComponent from 'components/string-with-component';
+import BeatmapJson from 'interfaces/beatmap-json';
+import BeatmapsetJson from 'interfaces/beatmapset-json';
+import LegacyMatchEventJson from 'interfaces/legacy-match-event-json';
+import Ruleset from 'interfaces/ruleset';
+import ScoreJson from 'interfaces/score-json';
+import UserJson from 'interfaces/user-json';
+import _ from 'lodash';
+import * as React from 'react';
+import { classWithModifiers } from 'utils/css';
+import { formatNumber } from 'utils/html';
+import { trans, transExists } from 'utils/lang';
+import GameHeader from './game-header';
+import Score from './score';
+
+interface TeamScores {
+  blue: number;
+  red: number;
+}
```

Comment:
This is a temporary place for this interface; it will be lifted up from here as the typescript conversion goes further.